### PR TITLE
combine calls from mitochondrial and snv variant calling subworflows

### PR DESCRIPTION
**Pull request by ramprasadn**
see title

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by Lucpen — COMMENTED** [review]
LGTM, just one question, have you try running with more than one sample?

**Comment by ramprasadn** [comment]
> LGTM, just one question, have you try running with more than one sample?

yes, and it works :)

File: nextflow.config [hunk 1]
```diff
@@ -18,6 +18,7 @@ params {
     igenomes_ignore            = false
     local_genomes              = null
     save_reference             = false
+    mt_contig_name             = 'MT'
```
Comment (jemten) [inline nextflow.config:21]: It's been a bit back and forth but I think the default should be running with grch38 and then the default name should be `chrM`

File: subworkflows/local/call_snv.nf [hunk 2]
```diff
@@ -18,9 +20,9 @@ workflow CALL_SNV {
 	    case_info            // channel: [ case_id ]
 
     main:
-       ch_versions   = Channel.empty()
+        ch_versions   = Channel.empty()
 
-       if (variant_caller == "deepvariant") {
+        if (variant_caller == "deepvariant") {
```
Comment (jemten) [inline subworkflows/local/call_snv.nf:25]: align the else if on row 31

**Review by Lucpen — APPROVED** [review]

**Review by jemten — COMMENTED** [review]
Just a comment that bcftools view will ost likely be the faster option. You can run it with the `-t ^MT` to exclude the mitochondria. But gatk works as well of course

File: nextflow.config (hunk 1 above)
Comment (ramprasadn) [reply nextflow.config:21]: Yeah, sure. While we are not running grch38 ourselves, I imagine by the time we deploy this in production we will be there (hopefully) 🤞🏻 😄

**Pull request merged**